### PR TITLE
VsIG-MySQL-CRUD

### PR DESCRIPTION
**Pull request by VsIG-official**

### CODE REVIEW

**Comment by mezgoodle** [comment]
> Nice to meet you, @VsIG-official. Thank you for creating a pull request.
> There are some tasks for you:
> 
> * [x]  Need at least one label
> * [x]  Need at least one assignee
> * [x]  Make a pull request draft at first
> * [x]  Need a milestone (No, I don't need it)
> 
> To close issue send comment "close", to reopen - "reopen", to merge - "merge"

Lol, edit bot's message....you are strong)

File: docs/Tables/MySQL/alchemy.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    place_from = 'ALTERAC PASS',
+    place_to = 'DRAGON SHRINE',
+    price = 200,
+    time = '2021-09-22 22:15:00',
+    car = car_Medivac.id
+)
+
+session.add(route_Dragon_Shrine)
+
+session.commit()
```
Comment (mezgoodle) [inline docs/Tables/MySQL/alchemy.py:59]: Don't you want to create class Engine and call methods from it?
For example:
```python
mysql_engine = Engine(user=user, password=password, localhost=localhost, db_name=db_name)
mysql_engine.create_tables()
"""
def create_tables(self):
    Base = declarative_base()
    Base.metadata.create_all(self.engine)
"""
mysql_engine.add([car_Medivac,...])
"""
def add(self, data: list):
    self.session.add(*data) # spread list, i dont remember operator
    self.session.commit()
"""
```
And everything for others methods form _CRUD_.
> I will write it too for SQLite, so you can wait and watch it
I see our code in future like this:
```python
data = mysql_engine.get(some_data)
sqlite_engine.add(data)
data = sqlite_engine.get(some_data)
postgre_engine.add(data)
```

Comment (VsIG-official) [reply docs/Tables/MySQL/alchemy.py:59]: ok

Comment (mezgoodle) [reply docs/Tables/MySQL/alchemy.py:59]: deprecated

**Review by mezgoodle — APPROVED** [review]

File: docs/Tables/MySQL/alchemy.py (hunk 1 above)
Comment (VsIG-official) [reply docs/Tables/MySQL/alchemy.py:59]: ok

**Pull request closed without merging**